Lib/sqlite3/test/dbapi.py

```diff
@@ -190,6 +193,8 @@ def test_open_uri(self):
         with sqlite.connect('file:' + TESTFN + '?mode=ro', uri=True) as cx:
             with self.assertRaises(sqlite.OperationalError):
                 cx.execute('insert into test(id) values(1)')
+        cx.close()
+        gc_collect()
```

Comment:
Why the falls to GC collect here? These should not be necessary

Reply:
I know; it's the same issue as in #24203. I'll try my best to reproduce this on my Mac.

Reply:
FYI, discovered #26475 while debugging this, but it does unfortunately not fix this issue.